security/middleware.py

```diff
@@ -861,14 +853,14 @@ def process_existing_session(self, request):
         start_time = session[self.START_TIME_KEY]
         last_activity_time = session[self.LAST_ACTIVITY_KEY]
 
-        logger.debug("Session %s started: %s" % (
-            session.session_key,
-            start_time,
-        ))
-        logger.debug("Session %s last active: %s" % (
-            session.session_key,
-            last_activity_time,
-        ))
+        logger.debug("Session %s started: %s",
+                     session.session_key,
```

Comment:
The first argument dictates indentation. All subsequent arguments have to match the indentation of the first argument.

If you want to indent with 4 spaces, the first argument needs to be on a new line.